docs/examples/read_write_files.md

```diff
@@ -0,0 +1,140 @@
+
+const text = readFile("./people.json");
+
+text.then((response) => console.log(response));
+
+/**
+ * Output:
+ *
+ * [
+ *   {"id": 1, "name": "John", "age": 23},
+ *   {"id": 2, "name": "Sandra", "age": 51},
+ *   {"id": 5, "name": "Devika", "age": 11}
+ * ]
+ */
+```
+
+The Deno standard library enables more advanced interaction with the filesystem
+and provides methods to read and parse files. The `readJson()` and
+`readJsonSync()` methods allow developers to read and parse files containing
+JSON. All these methods require is a valid file path string which can be
+generated using the `fromFileUrl()` method.
+
+In the example below the `readJsonSync()` method is used, for asynchronus
+execution use the `readJson()` method.
+
+Currently some of this functionality is marked as unstable so the `--unstable`
+flag is required along with the `deno run` command.
+
+**Command:** `deno run --unstable --allow-read read.ts`
+
+```js
+import { readJsonSync } from "https://deno.land/std@0.65.0/fs/mod.ts";
+import { fromFileUrl } from "https://deno.land/std@0.65.0/path/mod.ts";
+
+function readJson(path: string): object {
+  const file = fromFileUrl(new URL(path, import.meta.url));
+  return readJsonSync(file) as object;
+}
+
+console.log(readJson("./people.json"));
+
+/**
+ * Output:
+ *
+ * [
+ *   {"id": 1, "name": "John", "age": 23},
+ *   {"id": 2, "name": "Sandra", "age": 51},
+ *   {"id": 5, "name": "Devika", "age": 11}
+ * ]
+ */
+```
+
+## Write
+
+The Deno runtime API allows developers to write text to files via the
+`writeTextFile()` method. It just requires a file path and text string. On
+success the method will return an empty promise.
+
+To run the command the `--allow-write` flag must be supplied to the `deno run`
+command.
+
+**Command:** `deno run --allow-write write.ts`
+
+```js
+async function writeFile(path: string, text: string): Promise<void> {
+  return await Deno.writeTextFile(path, text);
+}
+
+const write = writeFile("./hello.txt", "Hello World!");
+
+write.then(() => console.log("File written to."));
+
+/**
+ * Output: File written to.
+ */
+```
+
+The Deno standard library makes available more advanced features to write to the
+filesystem. For instance it is possible to write an object literal to a JSON
+file.
+
+This requires a combination of the `ensureFile()`, `ensureFileSync()`,
+`writeJson()` and `writeJsonSync()` methods. In the example below the
+`ensureFileSync()` and the `writeJsonSync()` methods are used. The former checks
+for the existence of a file, and if it doesn't exist creates it. The latter
+method then writes the object to the file as JSON. If asynchronus execution is
+required use the `ensureFile()` and `writeJson()` methods.
+
+To execute the code the `deno run` command needs the unstable flag and both the
+write and read flags.
+
+**Command:** `deno run --allow-write --allow-read --unstable write.ts`
+
+```js
+import {
+  ensureFileSync,
+  writeJsonSync,
+} from "https://deno.land/std@0.65.0/fs/mod.ts";
```

Comment:
Can you change the imports to use `std@$STD_VERSION`? This special string is replaced in the frontend with the correct std version.